### PR TITLE
refactor: rename and restructure `/tools` folder

### LINKED ISSUE #470: todo: rename and restructure `/tools` folder
Suggestion:

- [ ] Rename `/tools` to `/tasks`
- [ ] Have each filename match up to the task name. E.g. `db:reset` task for `/tasks/db_reset.ts`.

### PR DESCRIPTION
**Pull request by rj2704**
Description

This PR fixes #470 

Notes for Reviewers
rename and restructure /tools folder

### CODE REVIEW

File: tasks/reset_kv.ts
Comment (iuioiua) [inline tasks/reset_kv.ts:1]: Can you please rename this to `db_reset.ts` to match the `db:reset` task name? Ditto for `db:seed` and `db:dump` tasks.

**Review by iuioiua — CHANGES_REQUESTED** [review]
Thanks for submitting, @rj2704! Few suggested changes.

File: tasks/reset_kv.ts
Comment (rj2704) [reply tasks/reset_kv.ts:1]: Sure @iuioiua

**Review by iuioiua — APPROVED** [review]
Excellent! LGTM! Thank you for your contribution, @rj2704.

**Comment by rj2704** [comment]
> Excellent! LGTM! Thank you for your contribution, @rj2704.

Thanks, @iuioiua looking forward to making more contributions.

**Pull request merged**